packages/oaf/src/commands/clan/skills.ts

```diff
@@ -131,7 +131,7 @@ export async function parseOldLogs() {
             skills,
           },
         });
-        return;
+        continue;
```

Comment:
I think it'd be easier to understand the logic here if we got rid of this continue and wrapped the update in an else since it makes the control flow a little more obvious. Up to you.